cfme/tests/control/test_basic.py

```diff
@@ -549,9 +514,7 @@ def test_assign_two_random_events_to_control_policy(control_policy, control_poli
 
 
 @pytest.mark.tier(2)
-@pytest.mark.uncollectif(lambda appliance, policy_class: (
-    policy_class in PHYS_POLICIES and appliance.version < "5.9"))
-@pytest.mark.meta(blockers=[BZ(1565576, forced_streams=["5.9"],
+@pytest.mark.meta(blockers=[BZ(1565576, forced_streams=["5.9", "5.10"],
```

Comment:
This bug was closed `NOTABUG`, can you check if this is needed.

Reply:
I played around a bit with this, this test is skipped due to that BZ in 5.9, and in 5.10 `BZ.blocks` is `False` but the test fails. I spent some time trying to debug this, but ultimately decided to fix it in a later PR. Would you like me to fix it in this one?

Reply:
@john-dupuy Add a TODO here for now and look into fixing it in another PR.